templates/article/article.tpl

```diff
@@ -123,6 +123,22 @@
 		{$pubIdPlugin->getPubIdDisplayType()|escape}: {if $pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}<a id="pub-id::{$pubIdPlugin->getPubIdType()|escape}" href="{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}">{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}</a>{else}{$pubId|escape}{/if}
 	{/if}
 {/foreach}
+{if $galleys}
+	{foreach from=$pubIdPlugins item=pubIdPlugin}
+		{foreach from=$article->getGalleys() item=galley name=galleyList}
```

Comment:
May as well reuse `$galleys` rather than calling `$article->getGalleys()` again. Not a big deal, but OTOH plugins that work with template variables will be able to alter this behavior more easily.